agile-development/stand-ups/social-question/readme.md

```diff
@@ -0,0 +1,39 @@
+# Social question of the day
+
+The social question of the day is an optional short question to follow the [three project questions](../readme.md) in the daily stand-up. It develops team cohesion and interpersonal trust over the course of an engagement by facilitating the sharing of chosen personal context.
+
+If the team decides to incorporate a social question-of-the-day into the stand-up, the questions can be selected by the facilitator or sourced from the attendees. The latter is faster and the former will likely result in more diverse questions.
+
+> Tip: having the stand-up facilitator role rotate lets the facilitator choose the social question without burdening any one team member.
```

Comment:
would add "facilitator role rotate **each sprint** lets the ..."

Reply:
Agreed, changed.